src/openzwave-driver.cc

```diff
@@ -58,6 +58,50 @@ namespace OZW {
 		emit_cb->Call(2, cbinfo, resource);
 	}
 
+	// ===================================================================
+	NAN_METHOD(OZW::UpdateOptions)
+	// ===================================================================
+	{
+		Nan::HandleScope scope;
+		CheckMinArgs(1, "info");
+
+		OZW* self = ObjectWrap::Unwrap<OZW>(info.This());
+		std::string ozw_userpath;
```

Comment:
Can you use ::std::string instead everywhere, some compiler complain about namespacing issues otherwise.